src/app/groups/group-by-id.component.html

```diff
@@ -1,7 +1,9 @@
 <ng-container *ngIf="state$ | async as state">
 
   <ng-container *ngIf="state.data?.group as group">
-    <alg-group-header [groupData]="state.data"></alg-group-header>
+    @if (state.data) {
```

Comment:
it seems to be covered by the previous line... so may be change the previous line `ng-container *ngIf=` with it.

Reply:
Fixed